website/source/docs/concepts/tokens.html.md

```diff
@@ -42,14 +42,53 @@ Finally, the `-policy` flag can be used to set the policies associated
 with the token. Learn more about policies on the
 [policies concepts](/docs/concepts/policies.html) page.
 
+<a name="ttls-and-leases" />
 ## Token Time-To-Live and Leases
 
-Every non-root token has a time-to-live (TTL) associated with it.  After the
-TTL is up, the token will no longer function. In addition to no longer
-functioning, Vault will revoke it.
+Every non-root token has a time-to-live (TTL) associated with it, which is
+the number of seconds the token is valid for, _measured from the moment it is
+created_. After the TTL is up, the token will no longer function, and
```

Comment:
Oh. OK, I see what you were saying. But this sort of shows the confusion: because of the lines below this comment, I thought you were talking about the TTL field when you do a token lookup, because that gets reset when you do a renew. Maybe it'd be better to simply take things even further and explain about the dual TTL / max TTL, but making it clear that when you do a vault token-lookup you're seeing the current TTL for the token, which may not be the same as the max?